sympy/functions/special/tests/test_delta_functions.py

```diff
@@ -80,19 +80,14 @@ def test_DiracDelta():
 
 
 def test_heaviside():
-    assert Heaviside(0).func == Heaviside
+    assert Heaviside(0,nan).func == Heaviside
```

Comment:
It is better to use unchanged for this e.g.:
https://github.com/sympy/sympy/blob/aa96b23bb98f00542101c08d8ea02ef83949310b/sympy/core/tests/test_arit.py#L170

Reply:
Never saw this before, thanks ;)